rollback/types.go

```diff
@@ -0,0 +1,64 @@
+package rollback
+
+import (
+	"github.com/metrumresearchgroup/pkgr/cran"
+	"github.com/metrumresearchgroup/pkgr/desc"
+	"github.com/metrumresearchgroup/pkgr/gpsr"
+	"github.com/metrumresearchgroup/pkgr/pacman"
+	"github.com/spf13/afero"
+	"github.com/thoas/go-funk"
+)
+
+type RollbackPlan struct {
+	AllPackages []string
+	NewPackages []string
+	UpdateRollbacks []pacman.UpdateAttempt
+	InstallPlan gpsr.InstallPlan
+	Library string
+}
+
+//Constructor
```

Comment:
these should have comments as they are exported - for example

```
// CreateRollbackPlan creates a rollback plan to....
```

Reply:
btw - does the goland linter not complain for not commenting on exported functions/methods?

Reply:
Looks like it doesn't, maybe that's a setting I can tweak somewhere.